<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>image/Youtube.png</normaloff>image/Youtube.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="btn_login">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>한수원 한돋움OTF</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_autologin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>96</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한울림OTF</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>자동로그인</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_id">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한돋움</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_password">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한돋움</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_face">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>한수원 한돋움OTF</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>얼굴인식</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_password_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한울림OTF</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_id_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>한수원 한울림OTF</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_login</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_login</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_face</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
